src/test/coverartutils_test.cpp

```diff
@@ -34,13 +34,15 @@ class CoverArtUtilTest : public MixxxTest, public CoverArtCache {
 
 const QString kCoverLocationTest("res/images/library/cover_default.png");
 const QString kTrackLocationTest(QDir::currentPath() %
-                                 "/src/test/id3-test-data/cover-test.mp3");
+                                 "/src/test/id3-test-data/cover-test-png.mp3");
 
 TEST_F(CoverArtUtilTest, extractEmbeddedCover) {
     const QString kTestPath(QDir::currentPath() % "/src/test/id3-test-data/");
     QImage cover;
     // We never need to acquire security tokens for tests since we don't run
     // them in a sandboxed environment.
+	
+    // TODO(XXX) we need to compare with a reference file to ensure that we are reading correct data
```

Comment:
Sure, it would be nice to have it (this is why I added a TODO), but what let me stay away from it is:
1. I don't have the reference png file that have been embedded in tracks. I will see if I can extract it.
2. I don't know sufficently well the various file formats to know how are image files embedded exactly. If there is any little alteration, the test could fail.
3. I'm affraid this could break CoverArtUtil::searchImage if we have more than one image file in the test data directory. There is strange image manipulation in this test (load an embedded png cover, save it as JPEG, and so on). It will need me to understand whole test to know if and how it will be impacted.
4. My goal was to add a jpeg embedded cover art test to help work out bug lp:1448746 and lp:1451832 not to refactor whole tests ;)

Nevermind, I will try adding this test.